packages/dataviews/src/dataviews-layouts/grid/index.tsx

```diff
@@ -386,6 +402,7 @@ function ViewGrid< Item >( {
 													config={ {
 														sizes: size,
 													} }
+													posinset={ index + 1 }
```

Comment:
I wonder if the value should only be passed value when `isInfiniteScroll` is true, because when regular pagination is active it'll be providing the incorrect value (e.g. on the second page it'll start from `1` again).

Reply:
Hmm if you're planning to use `posinset` when infinite scroll is disabled, then yes that would make sense to do. Otherwise in this PR `posinset` is only rendered when infinite scroll is on.

Reply:
I've found a bit of a problem with the aria attributes on the list layout: they don't play well at all with the `Composite` component we're using. I've removed them from list for now, though I haven't disabled infinite scroll on it. 

`Composit` is built on ariakit and the error occurs on that side. Basically any `posinset` value passed to `Composite.Row` makes it explode 😅

Reply:
😬 What's the error?

Might be worth checking the issues for ariakit to see if anyone has had the same - https://github.com/ariakit/ariakit/issues. I searched for posinset and saw nothing though.

Reply:
Not sure if you tried it, but you could try setting the prop on the `<div />` that the `Composite.Row` renders.